src/pages/trails/ballots/view/BallotView.vue

```diff
@@ -493,11 +497,6 @@ export default {
             q-spinner(size="3em")
 </template>
 <style lang="sass">
-.body-info
```

Comment:
This seems to be reverting [ this pr ](https://github.com/telosnetwork/app-telos-native/pull/87/files)